packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -225,6 +225,34 @@ export class Blockchain {
   async getChainTxStats (nBlocks?: number, blockHash?: string): Promise<ChainTxStats> {
     return await this.client.call('getchaintxstats', [nBlocks, blockHash], 'number')
   }
+
+  /**
+   * Get all in-mempool descendants for the given transaction as string[]
+   * @param {string} txId the transaction id in mempool
```

Comment:
```suggestion
   * @param {string} txId the transaction id
```
like how you write in https://github.com/JellyfishSDK/jellyfish/pull/1404/files#diff-401e60eaa7898f5ac9e4d982fcf34c68a2cd450a7a3c372d8243827ba6189c5bR239